machine/os/ubuntu.go

```diff
@@ -0,0 +1,37 @@
+package os
+
+import (
+	"strings"
+
+	"github.com/docker/machine/drivers"
+)
+
+func init() {
+	RegisterRuntime("ubuntu", &RegisteredRuntime{
+		Detect: UbuntuDetection,
+	})
+}
+
+func UbuntuDetection(d drivers.Driver) (*Runtime, error) {
+	out, err := sshCommand(d, "lsb_release -i | cut -f2 -d:")
```

Comment:
Relying on `lsb_release` to exist as a command is going to cause trouble down the road.  The current standard is `/etc/os-release`:
```console
$ for image in ubuntu:{precise,trusty,utopic,vivid} debian:{wheezy,jessie}; do (set -x && docker run --rm $image cat /etc/os-release); done
+ docker run --rm ubuntu:precise cat /etc/os-release
NAME="Ubuntu"
VERSION="12.04.5 LTS, Precise Pangolin"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu precise (12.04.5 LTS)"
VERSION_ID="12.04"
+ docker run --rm ubuntu:trusty cat /etc/os-release
NAME="Ubuntu"
VERSION="14.04.2 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.2 LTS"
VERSION_ID="14.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
+ docker run --rm ubuntu:utopic cat /etc/os-release
NAME="Ubuntu"
VERSION="14.10 (Utopic Unicorn)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.10"
VERSION_ID="14.10"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
+ docker run --rm ubuntu:vivid cat /etc/os-release
NAME="Ubuntu"
VERSION="15.04 (Vivid Vervet)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu Vivid Vervet (development branch)"
VERSION_ID="15.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
+ docker run --rm debian:wheezy cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 7 (wheezy)"
NAME="Debian GNU/Linux"
VERSION_ID="7"
VERSION="7 (wheezy)"
ID=debian
ANSI_COLOR="1;31"
HOME_URL="http://www.debian.org/"
SUPPORT_URL="http://www.debian.org/support/"
BUG_REPORT_URL="http://bugs.debian.org/"
+ docker run --rm debian:jessie cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 8 (jessie)"
NAME="Debian GNU/Linux"
VERSION_ID="8"
VERSION="8 (jessie)"
ID=debian
HOME_URL="http://www.debian.org/"
SUPPORT_URL="http://www.debian.org/support/"
BUG_REPORT_URL="https://bugs.debian.org/"
```